src/patternfly/components/Toolbar/examples/Toolbar.md

```
@@ -557,6 +557,252 @@ Several components in the following examples do not include functional and/or ac
+      {{> toolbar-group-search}}
+      {{> toolbar-group-filter}}
+      {{> toolbar-group-label-group}}
+      {{> toolbar-group-action-inline}}
+    {{/toolbar-expandable-content}}
+  {{/toolbar-content}}
+{{/toolbar}}
+```
+
+This toolbar will expand at the `md` container breakpoint (478px) instead of the default `lg` (702px).
+
+### Comparing container breakpoints
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)